hazelcast/src/test/java/com/hazelcast/test/HazelcastParallelClassRunner.java

```diff
@@ -149,12 +149,7 @@ public void run() {
 
         private final Properties globalProperties;
 
-        private final ThreadLocal<Properties> localProperties = new InheritableThreadLocal<Properties>() {
-            @Override
-            protected Properties initialValue() {
-                return init(new Properties());
-            }
-        };
+        private final ThreadLocal<Properties> localProperties = new ThreadLocal<Properties>();
```

Comment:
I will have a deeper look tomorrow, but there is nothing wrong with the TL here. it's a test code only, it's not something you deploy & undeploy. so here we should prefer simplicity.